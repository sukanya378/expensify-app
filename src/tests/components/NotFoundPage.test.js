import React from "react";
import { shallow } from "enzyme";
import NotFoundFage from "../../components/NotFound";


test('should render NotFoundFage', () => {
    const wrapper = shallow(<NotFoundFage />)
    expect(wrapper).toMatchSnapshot()
})