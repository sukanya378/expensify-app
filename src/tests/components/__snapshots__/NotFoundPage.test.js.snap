// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundFage 1`] = `
<div>
  Not found - Go to 
  <Link
    replace={false}
    to="/"
  >
    Home page
  </Link>
</div>
`;
